src/dash-table/components/ControlledTable/index.tsx

```diff
@@ -679,6 +679,17 @@ export default class ControlledTable extends PureComponent<ControlledTableProps>
             });
         }
 
+        // Adjust the height of rows in the row_selectable and row_deletable columns
+        if (fixed_columns || (row_selectable || row_deletable)) {
```

Comment:
Good thinking, escaping processing for no fixed_columns. No point wrapping row_selectable || row_deletable.